x/upgrade/go.mod

```diff
@@ -183,5 +183,6 @@ require (
 
 replace (
 	github.com/cosmos/cosmos-sdk => ../../.
+	cosmossdk.io/api => ../../api
```

Comment:
Can you use a pseudo version from main if needed? We'll tag api soon.

Reply:
You mean separate api change to merge into main 1st or my fork main?

Reply:
I see no api/ changes here, so if x/upgrade requires one, lets simply use the latest commit from main as pseudo version.

Reply:
you is right